packages/ic-certification/src/hash_tree/mod.rs

```diff
@@ -34,6 +34,7 @@ impl<Storage: AsRef<[u8]>> Label<Storage> {
     }
 
     /// The length of the output of [`Self::write_hex`]
+    #[cfg(feature = "serde")]
     fn hex_len(&self) -> usize {
```

Comment:
Unrelated change. This is unused unless the `serde` feature is enabled and was causing Cargo to throw a warning